recipes/si/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+import os
+
+
+class SiConan(ConanFile):
+    name = "si"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bernedom/SI"
+    description = "A header only c++ library that provides type safety and user defined literals \
+         for handling pyhsical values defined in the International System of Units."
+    topics = ("physical units", "SI-unit-conversion",
+              "cplusplus-library", "cplusplus-17")
+    exports_sources = "include/*", "CMakeLists.txt", "test/*", "doc/CMakeLists.txt", "doc/*.md"
```

Comment:
```suggestion
    exports_sources = "CMakeLists.txt"
    settings = "os", "arch", "compiler", "build_type"
```

+ add a `CMakeLists` "wrapper" file in the recipe